db/models.py

```diff
@@ -1 +1,25 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True, blank=True)
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, related_name="skills" , on_delete=models.CASCADE,)
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True, blank=True)
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE, related_name="players")
+    guild = models.ForeignKey(Guild, on_delete=models.SET_NULL, null=True, related_name="players")
+    creat_at = models.DateTimeField(auto_now_add=True)
```

Comment:
There is a typo in the field name `creat_at`. It should be `created_at` to follow the conventional naming pattern and ensure clarity.